projects/aws/image-builder/cmd/build_test.go

```diff
@@ -0,0 +1,127 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/aws/eks-anywhere-build-tooling/image-builder/builder"
+)
+
+func TestValidateSupportedHypervisor(t *testing.T) {
+	testCases := []struct {
+		testName     string
+		buildOptions builder.BuildOptions
+		wantErr      bool
```

Comment:
I always find more useful to include the error message in the test case
That way to verify you are returning an error where you expect it and not somewhere else